build.gradle

```diff
@@ -40,6 +40,7 @@ allprojects {
     compile 'com.signalfx.public:signalfx-codahale:0.0.47'
     compile group: 'org.apache.kafka', name: 'kafka_2.12', version: '2.3.1'
     compile group: 'org.apache.kafka', name: 'kafka-clients', version: '2.3.1'
+    compile 'org.eclipse.jetty:jetty-server:8.1.19.v20160209'
```

Comment:
This version of Jetty has severe security vulnerabilities which is why it was removed in the first place.

Reply:
Upgraded to 
```
// https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server
compile group: 'org.eclipse.jetty', name: 'jetty-server', version: '9.4.29.v20200521'
```